src/main/resources/alma/fix/contribution.fix

```diff
@@ -348,4 +348,4 @@ do list (path: "contribution[]", "var": "$i")
 end
 
 replace_all("contribution[].*.agent.label","<<|>>","")
-
+replace_all("contribution[].*.agent.altLabel[].*","<<|>>","")
```

Comment:
This seems not to work here:

https://github.com/hbz/lobid-resources/blob/showAnotherReplaceAllError/src/test/resources/alma-fix/990141342350206441.json#L136-L137

```
      "altLabel" : [ "Albertus", "Albertus", "Albertus", "Albert", "Albert", "Albert", "Albert", "Albertus Magnus", "Alberto", "Albert", "Albert", "Alberto", "Albertus", "Albert", "Velikii Albert", "Albert", "Albertus", "Albertus", "Albertus", "Albertus", "Albertus", "Albertus", "Albertus", "Albertus", "Albertus", "Albertus", "Petrus", "Petrus", "Albertus", "Albertus", "Alberto", "Alberto", "Albertus", "Albertus", "Albert", "Pseudo-Albertus", "Ps.-Albertus", "Pseudo-Albertus", "Ps.-Albertus", "Pseudo-Albertus Magnus", "Magnus, Albertus", "Magnus, Albertus", "Bolstadius, Albertus", "Bollstädt, Albert Graf <<von>>", "Bollstaedt, Albert Graf <<von>>" ]
```

Reply:
@blackwinter: is this the same error as in #1584 ?

Reply:
Yes, this is exactly the same issue.